eng/pipelines/templates/stages/archetype-sdk-tests.yml

```diff
@@ -61,13 +61,17 @@ parameters:
     - Name: NET_live_test_base
```

Comment:
Do you have plans to rename the stage templates as well? I know that will cause a huge change because it requires updating all the ci.yml/tests.yml files in the repo but if we are going for a consistent change across repos maybe it is worth it?

Reply:
I do, but my plan is to first investigate unifying the pipelines, because if we do that, then the stage rename is moot.